dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/CompanyIdentifier.kt

```diff
@@ -1,18 +0,0 @@
-package org.dataland.datalandbackend.model
-
-import com.fasterxml.jackson.annotation.JsonProperty
-import org.dataland.datalandbackend.model.enums.company.IdentifierType
-
-/**
- * --- API model ---
- * Class for defining the company identifiers as a part of company information
```

Comment:
Regarding clean up: I see there are still two interfaces directly in the model folder. They should be moved to the interface folder. The comments in both those interface are also not correct. I would further suggest to maybe introduce more subfolders. For example: model->company, model->datapoints,interface->datapoints. This should improve the structure of the code. We can also directly dicuss this.

Reply:
I also just noticed that SfdrGeneralGeneral is actually implementing the frameworkBase interface without using it.

Reply:
@HussainRoker started working on this. @HussainRoker can you drop a comment here once you are finished?

Reply:
last open point here is that: data class SfdrGeneralGeneral uses the frameworkbase but it does not actually implement it.

Reply:
Should be done now